modules/nginxvts/testdata/vts.json

```diff
@@ -0,0 +1,43 @@
+{

```

Comment:
Does that module have version? If so then we need to put data into `vX.Y.Z` dir, see https://github.com/netdata/go.d.plugin/tree/master/modules/elasticsearch/testdata/

Reply:
there's no version currently

Reply:
I see that nging module has versions https://github.com/vozlt/nginx-module-vts#version